src/services/motionPhotoService.ts

```diff
@@ -25,8 +26,9 @@ export const decodeMotionPhoto = async (
         } else if (zipFilename.startsWith('video')) {
             motionPhoto.videoNameTitle =
                 originalName + fileExtensionWithDot(zipFilename);
-            motionPhoto.video = await zip.files[zipFilename].async(
-                'uint8array'
+            const video = await zip.files[zipFilename].async('uint8array');
+            motionPhoto.video = await FFmpegService.convertLivePhotoToMP4(
```

Comment:
we should do it in the `convertForPreview`, as this function is also used when the user clicks to download the file and in that case, we want to download the  original file without conversion

Reply:
Oh right, didn't notice this.